### PR TITLE
Add Unicode / Python 3 update to 2018.3.0 release notes

### PR DESCRIPTION
**Pull request by terminalmage**

### CODE REVIEW

File: doc/topics/releases/2018.3.0.rst [hunk 1]
```diff
@@ -4,6 +4,14 @@
 Salt 2018.3.0 Release Notes - Codename Oxygen
 =============================================
 
+Unicode/Python 3 Compatibility Improvements
+-------------------------------------------
+
+Support for Unicode has been greatly improved, and many Python 3 compatibility
```
Comment (cachedout) [inline doc/topics/releases/2018.3.0.rst]: I don't think this comma is necessary.

File: doc/topics/releases/2018.3.0.rst [hunk 2]
```diff
@@ -4,6 +4,14 @@
...
+fixes have been made. We continue to work toward improving both, and welcome
```
Comment (cachedout) [inline doc/topics/releases/2018.3.0.rst]: Or this one.

**Comment by damon-atkins** [comment]
My suggestion

Unicode/International Language Support
This is the first release which supports International Language under Python 3 (limited support for Python 2).  i.e. Non-ASCII strings.  A common error has been `codec can't decode byte 0xcc in position 49`.  Ensure the salt master LANG is UTF-8 based which can be checked with `salt-master --versions-report` indicating a UTF-8 locale

Python 3 Support
Further improvements have been made for Python 3 support, this is the recommend release for using Python 3.


The other thought I had is for the salt master on startup sends to the logs `--versions-report` because startup scripts can have different settings to users.

**Comment by terminalmage** [comment]
I've rewritten the section I added to the release notes.

**Review by rallytime — APPROVED** [review]
👍

**Pull request merged**